components/_patterns/01-atoms/04-images/icons/_icon_sprite.scss

```diff
@@ -1,11 +1,10 @@
 @mixin svg-common {
-  background: url("svg/sprite.css.svg") no-repeat;
+    background: url("svg/sprite.css.svg") no-repeat;
```

Comment:
Do you have your editor set to use four spaces for tabs? Most places, we're using 2. It looks like there are a few other places you've used 4 though.